.github/workflows/deploy.yml

```diff
@@ -84,11 +112,11 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          release_name: Prerelease ${{ github.ref }}
-          tag_name: ${{ github.ref }}
-          body: Automatic release for ${{ github.ref }}
+          release_name: Release ${{ needs.calculate-version.outputs.version }}
+          tag_name: v${{ needs.calculate-version.outputs.version }}
+          body: Release ${{ needs.calculate-version.outputs.version }}
           draft: false
-          prerelease: true
+          prerelease: false
```

Comment:
Do we want to have automatic releases as "true" releases instead of pre-releases?